tests/test_saucedemo_autorization_and_order.py

```diff
@@ -0,0 +1,166 @@
+import unittest
+from webdriver_manager.chrome import ChromeDriverManager
+from selenium.webdriver.chrome.service import Service
+from selenium.webdriver.common.by import By
+from selenium import webdriver
+
+TARGET_URL = 'https://www.saucedemo.com'
+
+
+class TestSaucedemo(unittest.TestCase):
+
+    def setUp(self) -> None:
+        service = Service(ChromeDriverManager().install())
+        self.driver = webdriver.Chrome(service=service)
+        self.driver.set_window_size(1024, 768)
```

Comment:
@aigulmustafina this specific test suite will fail in CI/CD.
Please check a solution for the SetUp method here as a sample - https://github.com/BurhanH/TestAutomationCourse-python-ui-project/blob/main/tests/test_toolsqa_alerts.py#L19
Thanks